breadcrumble.gemspec

```diff
@@ -9,10 +9,9 @@ Gem::Specification.new do |gem|
   gem.homepage      = "https://github.com/ma2gedev/breadcrumble"
 
   gem.files         = `git ls-files`.split($\)
-  gem.executables   = gem.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
+  gem.executables   = gem.files.grep(%r{^exe/}).map{ |f| File.basename(f) }
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.name          = "breadcrumble"
-  gem.require_paths = ["lib"]
```

Comment:
It's removable

> The default value is "lib"

https://docs.ruby-lang.org/en/3.0.0/Gem/BasicSpecification.html#method-i-require_paths